test/models/project_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class ProjectTest < ActiveSupport::TestCase
+  def setup 
+    @user = build(:user)
+    @subject = Project.create(name: 'test', slug: 'this-should-be-unique', user: @user)
+    @subject.save
+  end
+
+  test 'belongs to #user' do
+    assert_kind_of(User, @subject.user)
+  end
+
+  test 'user has many projects' do
+    @new_project = Project.create(name: 'test', user: @user, slug: '123')
+    @another_new_project = Project.create(name: 'test', user: @user, slug: '456')
+    assert_equal(3, @user.projects.size)
+  end
+
+  test 'slugs must be unique' do
```

Comment:
This [helper method](https://github.com/TechforgoodCAST/fusebox-insights/blob/master/test/test_helper.rb#L20) may help with this test - [example usage](https://github.com/TechforgoodCAST/fusebox-insights/blob/master/test/models/unknown_test.rb#L35).